.github/workflows/ci.yml

```diff
@@ -168,3 +168,22 @@ jobs:
           file-types: '[".rs"]'
           exclude-folders: '["./examples/ios"]'
           exclude-files: '[]'
+
+  check-unused-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Installs cargo-udeps
+        run: cargo install cargo-udeps
+
+      - name: Install alsa and udev
+        run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev
+
+      - name: Run cargo udeps
+        run: cargo udeps
```

Comment:
From the last run of this job: `To find dependencies that are not used by any target, enable '--all-targets'.`
Since Bevy supports quite a few different targets, how about adding `--all-targets` here?

Reply:
Targets here means build targets (tests, examples, ...) not target triples. Using `--all-targets` would prevent detecting when regular dependencies could be `dev-dependencies`.

Reply:
Ah OK.
It could still be helpful though, right? E.g. it would have caught the latest change in #2455